docs/community/ecosystem.md

```diff
@@ -0,0 +1,59 @@
+# MXNet's Ecosystem
+
+Community contributions to MXNet have added many new valuable features and functionality to support use cases such as model serving & portability, easy and flexible APIs, and educational material like crash courses and online books. This ecosystem page lists the projects that use MXNet, teach MXNet, or augment MXNet in some way.
+
+
+## Highlighted Project
+
+[![Promo image](https://cdn-images-1.medium.com/max/800/1*PwIMdZM7tpt3rmcyhlk2FQ.png)](https://medium.com/apache-mxnet/announcing-keras-mxnet-v2-2-4b8404568e75)
+#### [Keras-MXNet v2.2 released!](https://medium.com/apache-mxnet/announcing-keras-mxnet-v2-2-4b8404568e75)
+
+## Contents
+
+* [Learning MXNet and other Educational Resources](#learning-mxnet-and-other-educational-resources)
+* [New MXNet APIs](#new-mxnet-apis)
+* [Toolkits to Extend MXNet](#toolkits-to-extend-mxnet)
+* [Model Serving](#model-serving)
+* [Model Zoos](#model-zoos)
+* [MXNet as a Backend](#mxnet-as-a-backend)
+* [Contributions](#contributions)
+
+## Learning MXNet and other Educational Resources
+
+* [Gluon 60 Minute Crash Course](https://gluon-crash-course.mxnet.io/) - deep learning practitioners can learn Gluon quickly with these six 10-minute tutorials.
+* [The Straight Dope](https://gluon.mxnet.io/) - a series of notebooks designed to teach deep learning using the Gluon Python API for MXNet.
+
+
+## New MXNet APIs
```

Comment:
Fair point. We'd probably cycle out Gluon and the other APIs as they mature and have a lot of references on the site. 
I said new because I didn't want to list ALL of the APIs... just highlight the new ones.

For news and feeding the home page I was thinking of a specific news page, examples page, and tutorials page. Each would provide a feed fro the home page. Krishnan is working on the redesign to support that.

Reply:
If the argument to highlight an API is "it's new" then it should go into news. Otherwise all APIs are equal.
Given that APIs already have their own very noticeable place at the top of the page, I'm not sure if it's necessary to highlight APIs in MXNet again here.

Reply:
Not all newly introduced APIs have top level or second level navigation, such as Clojure and the Scala Infer API. I could remove Gluon, but frankly, I'd prefer to highlight Gluon here (and other places throughout the site) than have it as a separate top-level navigation item. That makes Gluon very much not equal, and IMO, it's confusing to new users.

Discoverability is important. Having more references and cross-links is good. People will be able to items listed in a different context and with a description. With just a navigation item I might never click on ONNX. But shown in the context of models, model zoo, or model server, I might. The same goes for the other topics like Gluon. The word "Gluon"  is used with the NLP and CV toolkits and as a Python API, and just by scanning the page, the user can see a better picture of what they can do with it.

BTW, I'm not saying no... just throwing out some counter arguments to consider.

Reply:
Should it be done in news instead?

Reply:
If you prefer you can link to the news.